src/pages/resetForm.js

```diff
@@ -0,0 +1,80 @@
+import { Form, Input, Button } from 'antd'
+import { UserOutlined } from '@ant-design/icons'
+import { Link, useNavigate } from 'react-router-dom'
+import { routes } from '../config/routes'
+import { httpsService } from '../utils/https.service'
+import { notificationService } from '../utils/notification.service'
+import '../styles/resetForm.scss'
+
+export const ResetForm = () => {
+  const navigate = useNavigate()
+
+  const handleReset = async (body) => {
+    try {
+      const response = await httpsService.post('/user/reset', body)
+
+      if (response) {
+        notificationService.openNotification(
+          'info',
+          '',
+          'New password was sent to your email!',
+          10
+        )
+        navigate(routes.login)
+      } else {
+        notificationService.openNotification(
+          'error',
+          '',
+          'This email is not valid or not supported',
+          10
+        )
+      }
```

Comment:
we discussed that we will show success message in any case. This is potential security issue. Do we need this if/else statement?